lib/api/routes.js

```diff
@@ -73,4 +73,11 @@ module.exports = [
         extensions: { crr: ['failed'] },
         method: 'retryFailedCRR',
     },
+    {
+        httpMethod: 'GET',
+        category: 'monitoring',
+        type: 'monitoring',
```

Comment:
do you need the `type` field? If not I would rather leave it unset for now until we have a good use for it.

Reply:
ya, maybe we can leave it empty as we aren't using it. I think the focus was to keep it as uniform with other routes as possible. Just to note also for healthcheck, we do not use the `type` field, so I could also set that to empty as well in a refactor later